artifacts/mainnet/ics/assessment/engagement/main.py

```diff
@@ -0,0 +1,272 @@
+    if total_votes_count > REQUIRED_ARAGON_VOTES:
+        return scores["aragon-vote"]
+    return 0
+
+
+def galxe_scores(addresses: Iterable[str]) -> int:
+    api_url = "https://graphigo.prd.galaxy.eco/query"
+    lido_space_id = 22849
+    query = """
+        query($spaceId: Int, $cursor: String) {
+      space(id:$spaceId) {
+        id
+        name
+        loyaltyPointsRanks(first:100,cursorAfter:$cursor)
+        {
+          pageInfo{
+            hasNextPage
+            endCursor
+          }
+          edges {
+            node {
+              points
+              address {
+                username
+                address
+              }
+            }
+          }
+        }
+      }
+    }
+    """
+
+    def fetch_all_items():
+        cursor = None
+        all_items = []
+        while True:
+            variables = {"spaceId": lido_space_id, "cursor": cursor}
+            response = requests.post(
+                api_url,
+                json={"query": query, "variables": variables},
+                headers={"Content-Type": "application/json"}
+            )
+            response.raise_for_status()
+            data = response.json()['data']['space']['loyaltyPointsRanks']
+
+            for edge in data['edges']:
+                all_items.append(edge['node'])
+
+            page_info = data['pageInfo']
+            if not page_info['hasNextPage']:
+                break
+            cursor = page_info['endCursor']
+        return all_items
+
+    all_items = fetch_all_items()
+    addr_to_points = {item["address"]["address"].lower(): item["points"] for item in all_items}
+
+    score = 0
+    for address in addresses:
+        point = addr_to_points.get(address, 0)
+        if point > 10:
+            score = scores["galxe-score-above-10"]
+            # max score, no need to check further
+            print(f"    Found {point} Galxe score for address {address}")
+            return score
+        elif 4 <= point <= 10:
+            print(f"    Found {point} Galxe score for address {address}")
+            score = scores["galxe-score-4-10"]
+    return score
+
+
+def gitpoap(addresses: Iterable[str]) -> int:
+    url = "https://public-api.gitpoap.io/v1"
+
+    with open(current_dir / "gitpoap_events.csv", "r") as f:
+        reader = csv.DictReader(f)
+        gitpoap_events = {row["ID"]: row["Name"] for row in reader}
+    s = requests.Session()
+    a = requests.adapters.HTTPAdapter(max_retries=3)
+    s.mount('https://', a)
+
+    final_score = 0
+    for event_id, event_name in gitpoap_events.items():
+        response = s.get(f"{url}/gitpoaps/{event_id}/addresses")
+        response.raise_for_status()
+
+        poap_holders = response.json().get("addresses", [])
+        if any(address.lower() in poap_holders for address in addresses):
+            print(f"    Found GitPoap for event '{event_name}'")
+            final_score =  scores["git-poap"]
```

Comment:
```suggestion
            return scores["git-poap"]
```

If I understood correctly.